cppcache/static/CMakeLists.txt

```diff
@@ -16,21 +16,21 @@
 cmake_minimum_required(VERSION 3.4)
 project(apache-geode-static LANGUAGES CXX)
 
-add_library(apache-geode-static STATIC ${SOURCES_ALL})
+add_library(${PROJECT_NAME} STATIC ${SOURCES_ALL})
```

Comment:
Don't use ${PROJECT_NAME} like this. It breaks integration with CLion and other CMake integrations. Revert these changes.

Reply:
Done